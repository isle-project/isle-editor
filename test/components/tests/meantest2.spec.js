// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import MeanTest2 from '@isle-project/components/tests/meantest2';
import heartdisease from '../fixtures/heartdisease.json';


// TESTS //

describe( '<MeanTest2 />', function test() {
	it( 'renders test result', () => {
		const output = renderer.create( <MeanTest2
			data={heartdisease}
			x="Cost"
			group="Gender"
		/> );
		expect( output ).toMatchSnapshot();
	});
});
