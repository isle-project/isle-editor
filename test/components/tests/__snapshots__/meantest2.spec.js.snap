// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MeanTest2 /> renders test result 1`] = `
<div
  style={
    {
      "overflowX": "auto",
      "width": "100%",
    }
  }
>
  <span
    className="title"
  >
    hypothesis-test-between
    :
  </span>
  <div />
  <span>
     
    vs
     
  </span>
  <div />
  <pre>
    Welch two-sample t-test

Alternative hypothesis: True difference in means is not equal to 0

    pValue: 0.5789
    statistic: 0.5555
    df: 326.677
    95% confidence interval: [-749.7543,1339.8251]


    <br />
    sample-mean-in-group
     "
    Female
    ": 
    2867.35
    <br />
    sample-mean-in-group
     "
    Male
    ": 
    2572.314
  </pre>
</div>
`;
